bip-taro.mediawiki

```diff
@@ -205,7 +205,7 @@ The Taro asset tree is a core component of the system. An asset tree is an
 MS-SMT commitment to a series of asset identifiers, grouped by their genesis
 asset ID. A given Taro asset tree is composed of two nested MS-SMT instances:
 
-# The first level maps an <code>asset_id</code> or <code>asset_key_family</code> to a sub-tree root hash of a given asset.
```

Comment:
Thought we had already done this? 🤔 cc @jharveyb

Reply:
Back then nope, but should be able to rebase and take away these three commits now 🫡